docs/source/discovery-overview.rst

```diff
@@ -30,6 +30,11 @@ The **discovery service** improves this process by having the peers compute
 the needed information dynamically and present it to the SDK in a consumable
 manner.
 
+Starting with v2.4, the :doc:`gateway` interacts with service discovery in order to determine
+which peers are required to endorse a transaction, and which orderer nodes to send
+the transaction.  Consequently, client applications written using the new gateway
```

Comment:
"the transaction to."

Reply:
"the new Fabric Gateway" ... (i.e. let's be formal about it at first because it's a new thing. Over time it can revert to more informal "gateway")

Reply:
test